.azure-pipelines/crank.yml

```diff
@@ -41,10 +41,10 @@ jobs:
       version: 3.1.x
 
   - task: UseDotNet@2
-    displayName: install dotnet core sdk 5.0
+    displayName: install dotnet core sdk 5.0.103
```

Comment:
This was introduced here: https://github.com/DataDog/dd-trace-dotnet/pull/1352
However, this is not needed. IMO a better fix one was already merged in OTel repo: https://github.com/open-telemetry/opentelemetry-dotnet-instrumentation/pull/120

@andrewlock @tonyredondo 
The corresponding fix in DD was not merged: https://github.com/DataDog/dd-trace-dotnet/pull/1353

Reply:
Yes, there was a bit of discussion around that, will likely be merged soon. 

However, I think the fix introduced in https://github.com/DataDog/dd-trace-dotnet/pull/1352 is still worth keeping. It was the only pipeline using a different SDK version to the others.

Reply:
👍